src/crimson/os/seastore/root_block.h

```diff
@@ -97,10 +97,12 @@ struct RootBlock : CachedExtent {
 
   complete_load_ertr::future<> complete_load() final {
     assert(0 == "Root is only written via deltas");
+    throw std::logic_error("Root is only written via deltas");
```

Comment:
How about `ceph_abort_msg()`?

Reply:
@rzarzynski thanks! updated and repushed.